src/sass/_shame.scss

```diff
@@ -138,3 +138,54 @@ body .row {
 .csp-inline-patch-logout {
   padding: 2em 0;
 }
+
+// /education/apply/ uses this expander button for navigation
+// Where else can I put this? Also, it could use a better name.
```

Comment:
If this is something other teams might use at some point, we don't want to limit it to just this page anyway, right? But I agree with the last point, that we can probably leave it in _shame.scss for now. But I would suggest creating a ticket for us to share this UI pattern with the design team and ask if they would find this useful in other places. If so, we can then figure out how to make this extensible at that point. Just so we don't forget to come back to that :)